ide/api.lsp/nbproject/project.xml

```diff
@@ -52,6 +52,15 @@
                         <specification-version>1.77</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.spi.editor.hints</code-name-base>
```

Comment:
This dependency is increasing [weight of the api.lsp module](http://wiki.apidesign.org/wiki/HeavyWeight). How much? I see five editor related libraries. Can such additional height be avoided?

I lived under the impression that the goal of LSP isn't to expose NetBeans Editor API, but rather abstract away from it. Bringing these libraries here via transitive dependencies seems to go against such goal.

@dbalek - right now there is one dependency on editor API in the `api.lsp` module - but hopefully it could be removed by replacing it with direct call to `(String)doc.getProperty("mimeType")`.